cmd/m3db-operator/main.go

```diff
@@ -98,10 +97,6 @@ func main() {
 
 	defer logger.Sync()
 
-	logger.Info("Go", zap.Any("VERSION", runtime.Version()))
```

Comment:
Why get ride of this?

Reply:
Since I added the m3x build reporter we get all of this info for free in a startup log from there

Reply:
nice.